src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -0,0 +1,122 @@
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="Black" />
+                    </Style>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="Light">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="White" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="Black" />
+                    </Style>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="{ThemeResource SystemColorButtonTextColor}" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="{ThemeResource SystemColorWindowColor}" />
+                    </Style>
+                </ResourceDictionary>
+            </ResourceDictionary.ThemeDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <StackPanel Orientation="Vertical" Style="{ThemeResource SearchBoxBackground}">
+        <StackPanel Orientation="Horizontal" HorizontalAlignment="Right">
+            <Button x:Name="CloseButton" AutomationProperties.Name="Close" Padding="0" Click="_CloseClick">
+                <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE711;" FontSize="15" Foreground="White"/>
+            </Button>
+        </StackPanel>
+
+        <StackPanel x:Name="TextBoxStackPanel" Margin="0, 3, 0, 3" Orientation="Horizontal">
+            <TextBox  x:Name="TextBox" AutomationProperties.Name="Text" PlaceholderText="Search"
+                KeyDown = "QuerySubmitted"
+                Width="160" Margin="0,0,1,0" HorizontalAlignment="Left" VerticalAlignment="Center">
+            </TextBox>
+
+            <ToggleButton x:Name="SetGoBackwardButton" AutomationProperties.Name="SetGoBackward" HorizontalAlignment="Right"

```

Comment:
```suggestion
            <ToggleButton x:Name="SetGoBackwardButton" AutomationProperties.Name="Set Search Backward" HorizontalAlignment="Right"
```
The AutomationProperties.Name should be the human-readable name which accessibility tools can read aloud to users.

Also, these strings should be localized if the app is localized into other languages.  If Terminal is localized at all, you might want to consider x:Uid (https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/x-uid-directive) as the way to set these names and the PlaceholderText of the TextBox.

Reply:
AutomationProperties.Names changed. Let me sync with Dustin about the localization issue.